blocks/monitoring/aggregation.py

```diff
@@ -138,11 +138,62 @@ def get_aggregator(self):
         return aggregator
 
 
-def mean(numerator, denominator=1.):
+def mean(numerator, denominator=1., name=None):
     """Mean of quantity (numerator) over a number (denominator) values."""
     variable = numerator / denominator
     variable.tag.aggregation_scheme = Mean(numerator, denominator)
-    variable.name = numerator.name
+    if name is not None:
```

Comment:
I am tempted to agree that this is a reasonable solution, but I am 99% sure @bartvm will hate us for that. Because this is a non-standard extension of Python core, and our need for it is barely justifiable by this additional single line `per_token_cost.name = 'per-token-cost'` one has to write each time.